samples/Server/src/lib/envHelper.ts

```diff
@@ -4,12 +4,28 @@
 import * as fs from 'fs';
 import * as path from 'path';
 const appSettingsPath = path.join(__dirname, '../../appsettings.json');
+let appSettings: {
+  ResourceConnectionString: string;
+  EndpointUrl: string;
+  AdminUserId: string;
+  AzureBlobStorageConnectionString: string;
+};
 if (!fs.existsSync(appSettingsPath)) {
-  throw new Error(
-    'No appsettings.json found. Please provide an appsettings.json file by copying appsettings.json.sample and removing the .sample extension'
-  );
+  if (
+    !(
+      process.env['ResourceConnectionString'] ||
+      process.env['EndpointUrl'] ||
```

Comment:
do we want to try extract endpointUrl from ResourceConnectionString?

Reply:
let me do that in seperate PR, as getting this working is time sensetive